packages/wallet/backend/src/account/service.ts

```diff
@@ -291,16 +296,26 @@ export class AccountService implements IAccountService {
   }
 
   public async createDefaultAccount(
-    userId: string
+    userId: string,
+    type: DefaultAccountType = DefaultAccountType.EUR
   ): Promise<Account | undefined> {
+    let assetCode = 'EUR'
```

Comment:
'replace the default EUR account and payment pointer creation with a USD account and USD payment pointer' -> from story, I understand here, that we can remove EUR default account and PP creation, and just leave in USD creation